test/unit/solver/mc/test_triangular.cpp

```diff
@@ -97,3 +156,24 @@ TYPED_TEST(TriangularSolverLocalTest, Correctness) {
     }
   }
 }
+
+TYPED_TEST(TriangularSolverDistributedTest, Correctness) {
+  SizeType m, n, mb, nb;
+
+  for (const auto& comm_grid : {this->commGrids()[0]}) {
+    for (auto diag : blas_diags) {
+      for (auto op : blas_ops) {
+        for (auto side : blas_sides) {
+          for (auto uplo : blas_uplos) {
```

Comment:
I propose as solution ot the above comment to check here if the case is implemented:
In this case:
```
// Currently only Left Lower Notrans case is implemented
if (!(op ==NoTrans && side == Left && uplo ==Lower))
  continue;
```

Reply:
Good suggestion, thanks.